Exception in thread "pool-1-thread-2" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-7" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-6" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-5" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-4" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-3" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-10" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-8" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-17" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-1" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-9" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-18" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-11" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-16" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-22" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-19" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-21" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-20" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-29" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExec[INFO] Tests run: 25, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.712 s - in zipkin2.CallTest
[INFO] Running zipkin2.SpanTest
[INFO] Tests run: 11, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.091 s - in zipkin2.SpanTest
[INFO] Running zipkin2.AnnotationTest
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.005 s - in zipkin2.AnnotationTest
[INFO] Running zipkin2.codec.KryoTest
[INFO] Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.073 s - in zipkin2.codec.KryoTest
[INFO] Running zipkin2.codec.SpanBytesEncoderTest
[INFO] Tests run: 34, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.043 s - in zipkin2.codec.SpanBytesEncoderTest
[INFO] Running zipkin2.codec.EncodingTest
[INFO] Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.002 s - in zipkin2.codec.EncodingTest
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   EndpointTest.ip_ipv6_addr_compatIpv4:154 expected:<"43.0.[192].2"> but was:<"43.0.[-64].2">
[INFO] 
[ERROR] Tests run: 221, Failures: 1, Errors: 0, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] Zipkin (Parent) .................................... SUCCESS [  5.097 s]
[INFO] Zipkin v2 .......................................... FAILURE [ 11.580 s]
[INFO] Zipkin ............................................. SKIPPED
[INFO] Zipkin UI .......................................... SKIPPED
[INFO] Zipkin JUnit ....................................... SKIPPED
[INFO] Guava support library .............................. SKIPPED
[INFO] Auto Configuration ................................. SKIPPED
[INFO] Auto Configuration: Ui ............................. SKIPPED
[INFO] Storage ............................................ SKIPPED
[INFO] Storage: Cassandra ................................. SKIPPED
[INFO] Auto Configuration: Cassandra Storage .............. SKIPPED
[INFO] Storage: Cassandra (Zipkin V2) ..................... SKIPPED
[INFO] Auto Configuration: Cassandra 3 Storage ............ SKIPPED
[INFO] Storage: Elasticsearch (V2) ........................ SKIPPED
[INFO] Storage: Elasticsearch (HTTP) ...................... SKIPPED
[INFO] Auto Configuration: Elasticsearch Storage (HTTP) ... SKIPPED
[INFO] Auto Configuration: Elasticsearch Storage (AWS) .... SKIPPED
[INFO] Storage: MySQL ..................................... SKIPPED
[INFO] Auto Configuration: MySQL Storage .................. SKIPPED
[INFO] Zipkin Collectors .................................. SKIPPED
[INFO] Collector: Kafka ................................... SKIPPED
[INFO] Auto Configuration: Kafka Collector ................ SKIPPED
[INFO] Collector: RabbitMQ ................................ SKIPPED
[INFO] Auto Configuration: RabbitMQ Collector ............. SKIPPED
[INFO] Auto Configuration: Prometheus Metrics ............. SKIPPED
[INFO] Zipkin Server ...................................... SKIPPED
[INFO] Benchmarks ......................................... SKIPPED
[INFO] Collector: Scribe .................................. SKIPPED
[INFO] Collector: Kafka 0.10+ ............................. SKIPPED
[INFO] Auto Configuration: Kafka 0.10 Collector ........... SKIPPED
[INFO] Auto Configuration: Scribe Collector ............... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 18.244 s
[INFO] Finished at: 2020-02-16T15:09:05+01:00
[INFO] Final Memory: 33M/81M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.21.0:test (default-test) on project zipkin: There are test failures.
[ERROR] 
[ERROR] Please refer to /scratch/dginelli/workspace/repairnator-repairnator-experiments-openzipkin-zipkin-367183201-20180416-162837-firstCommit/zipkin2/target/surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date]-jvmRun[N].dump, [date].dumpstream and [date]-jvmRun[N].dumpstream.
[ERROR] -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.21.0:test (default-test) on project zipkin: There are test failures.

Please refer to /scratch/dginelli/workspace/repairnator-repairnator-experiments-openzipkin-zipkin-367183201-20180416-162837-firstCommit/zipkin2/target/surefire-reports for the individual test results.
Please refer to dump files (if any exist) [date]-jvmRun[N].dump, [date].dumpstream and [date]-jvmRun[N].dumpstream.
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:212)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:863)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:288)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:199)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: org.apache.maven.plugin.MojoFailureException: There are test failures.

Please refer to /scratch/dginelli/workspace/repairnator-repairnator-experiments-openzipkin-zipkin-367183201-20180416-162837-firstCommit/zipkin2/target/surefire-reports for the individual test results.
Please refer to dump files (if any exist) [date]-jvmRun[N].dump, [date].dumpstream and [date]-jvmRun[N].dumpstream.
	at org.apache.maven.plugin.surefire.SurefireHelper.throwException(SurefireHelper.java:240)
	at org.apache.maven.plugin.surefire.SurefireHelper.reportExecution(SurefireHelper.java:112)
	at org.apache.maven.plugin.surefire.SurefirePlugin.handleSummary(SurefirePlugin.java:354)
	at org.apache.maven.plugin.surefire.AbstractSurefireMojo.executeAfterPreconditionsChecked(AbstractSurefireMojo.java:1008)
	at org.apache.maven.plugin.surefire.AbstractSurefireMojo.execute(AbstractSurefireMojo.java:854)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:207)
	... 20 more
[ERROR] 
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :zipkin
executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-45" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-46" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-47" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-48" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-36" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-50" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-51" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-52" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-53" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-42" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-44" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "pool-1-thread-43" java.lang.IllegalStateException: Already Executed
	at zipkin2.Call$Base.enqueue(Call.java:387)
	at zipkin2.CallTest.lambda$concurrent_executesOrSubmitsOnce$5(CallTest.java:132)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
